src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -84,7 +84,39 @@ public async Task DeleteNotFound()
         }
 
         [Test]
-        public async Task Delete()
+        public async Task DeleteSetting()
+        {
+            var connectionString = Environment.GetEnvironmentVariable("AZ_CONFIG_CONNECTION");
```

Comment:
Extract a method.

Reply:
Wouldn't it be best to extract this and the assert not null into a before test setup since the code pattern in consistent in all the tests?